test/el-test.js

```diff
@@ -126,3 +127,129 @@ describe('yo polymer:el --path test', function () {
     );
   });
 });
+
+describe('yo polymer:el TDD test', function () {
+
+  before(function (done) {
+    helpers.run(path.join(__dirname, '../el'))
+      .inDir(path.join(__dirname, './tmp'))
+      .on('ready', function (generator) {
+        var js = "WCT.loadSuites(['my-greeting-basic.html', 'my-list-basic.html']);";
+        fs.outputFileSync(path.join(__dirname, './tmp/app/test/index.html'), js);
```

Comment:
I tried that originally actually but it doesn't seem to work the way you describe.
It seems like the files generated in the first setup phase are deleted after the second one is run. 

Notice the `tmp` directory before it hits the second call to `helpers.run`
![image](https://cloud.githubusercontent.com/assets/324928/8962854/e28c8a18-35ea-11e5-8889-7e2c69d91375.png)

Now look what the `tmp` directory contains in the first test
![image](https://cloud.githubusercontent.com/assets/324928/8963042/fbe74272-35eb-11e5-8bad-804891a5d48c.png)

I've never written test for yeoman before and I actually hacked at it for quite a while before I gave up and went with the solution in this pull request.